explanation/faq.md

```diff
@@ -10,11 +10,19 @@ Not at the moment. But [Canonical](https://www.canonical.com), the company who
 
 The v1 series will be maintained, improved and bug-fixed for the foreseeable future and backward compatibility is guaranteed.
 
-## How does Dqlite behave during conflict situations?
+## How does Dqlite behave during conflict situations? Does Raft select a winning WAL to write and any others in-flight writes are aborted?
```

Comment:
I think the original, short heading is better.

If the clarification is needed, then it belongs to one of the paragraphs below.

Reply:
This comment made me realize that the word "majority" doesn't appear in the FAQ outside this paragraph! Good idea to make it more prominent.